zookeeper-server/src/main/java/org/apache/zookeeper/server/backup/BackupPoint.java

```diff
@@ -42,12 +43,20 @@ public BackupPoint(long logZxid, long snapZxid) {
    */
   public long getLogZxid() { return logZxid; }
 
+  public void setLogZxid(long logZxid) {
+    this.logZxid = logZxid;
+  }
+
   /**
    * Get the starting zxid of the latest backed up snap
    * @return the starting zxid of the latest backed up snap
    */
   public long getSnapZxid() { return snapZxid; }
 
+  public void setSnapZxid(long snapZxid) {
+    this.snapZxid = snapZxid;
+  }
+
   /**
    * Get the starting timestamp of the latest backed up timetable backup file
    * @return the starting timestamp of the latest backed up timetable backup file
```

Comment:
The ending timestamp is used in the code I believe. Let's update this java doc comment? Also don't forget to add param description for timetable in BreakPoint constructor.